### PR TITLE
michelson: add file inclusion functionality

### PR DESCRIPTION
**Pull request by sskeirik**
Currently, relative paths are interpreted relative to `./kmich` and not to the file which we are importing from.

This is not ideal, but I'm not sure of the best way to fix this.

My thought is we can merge this now and come back and fix this issue later. Thoughts?

Fixes: runtimeverification/firefly-michelson#16

### CODE REVIEW

**Comment by ehildenb** [comment]
Hmmmm, that's unfortunate. Maybe for the `clarity-driver.md` we can pass in (as configuration variables) (i) the current directory, and (ii) the path to the input file that is being used?

**Comment by sskeirik** [comment]
@nishantjr I am trying to add an additional input to the program (the definition directory) and it works for the llvm interpreter but the symbolic driver isn't working right. Could you take a look at this or could we discuss this tomorrow at the standup? Thanks.

File: kmich [hunk 1]
```diff
@@ -21,7 +21,7 @@ if [[ ! -f "${k_release_dir}/bin/kompile" ]]; then
     fi
 fi
 
-export PATH="$scriptpath/lib:$k_release_dir/bin:$PATH"
+PATH="$scriptpath/lib:$k_release_dir/bin:$PATH"
```
Comment (ehildenb) [inline kmich:24]: ????

Comment (sskeirik) [reply kmich:24]: I want to export the PATH only once after getting the location of `parser_PGM`. The export moved farther down the file.

**Comment by sskeirik** [comment]
@ehildenb This is ready for review. I updated the file inclusion functionality so that:

1. it only applies to the LLVM backend
2. it is always relative to the input test file

**Review by nishantjr — APPROVED** [review]
LGTM

**Pull request merged**